interfaces/Portalicious/src/app/pages/project/project-registrations/project-registrations.page.html

```diff
@@ -17,6 +17,69 @@
     >
       <div table-actions>
         <p-buttonGroup>
+          @if (
+            authService.hasPermission({
+              projectId: projectId(),
+              requiredPermission:
+                PermissionEnum.RegistrationStatusMarkAsValidatedUPDATE,
+            }) && project()?.validation
+          ) {
+            <p-button
+              label="Validate"
+              i18n-label
+              icon="pi pi-check-circle"
+              rounded
+              outlined
+              (click)="changeStatus(RegistrationStatusEnum.validated)"
+            />
+          }
+          @if (
+            authService.hasPermission({
+              projectId: projectId(),
+              requiredPermission:
+                PermissionEnum.RegistrationStatusIncludedUPDATE,
+            })
+          ) {
```

Comment:
To avoid the duplication in the ifs I'm wondering if it might be nice to create a method you could call like

`@if (canChangeStatus(RegistrationStatusEnum.included) {`